src/utils/__tests__/makeReactNativeConfig.test.js

```diff
@@ -62,41 +62,47 @@ describe('makeReactNativeConfig', () => {
 });
 
 describe('injects polyfill into different entries', () => {
-  const fakePolyfillPath = 'path/to/polyfill.js';
-
   test('entry is a string', () => {
     const userEntry = './src/index.js';
-    const generatedEntry = injectPolyfillIntoEntry(userEntry, fakePolyfillPath);
+    const generatedEntry = injectPolyfillIntoEntry({
+      entry: userEntry,
+      root: path.resolve('src/utils/__tests__/fixtures'),
+    });
 
-    expect(generatedEntry.length).toBe(2);
-    expect(generatedEntry[0]).toBe(fakePolyfillPath);
+    // $FlowFixMe
```

Comment:
We could get rid of $FlowFixMe by checking that it's an array and throw instead (this may be confusing for future readers)

Reply:
Good point!